dhis-2/dhis-web/dhis-web-importexport/src/main/webapp/dhis-web-importexport/metadataDependencyExport.vm

```diff
@@ -24,8 +24,8 @@
     <td>$i18n.getString( "format" )</td>
     <td>
       <select id="format" style="width:280px">
-        <option value="json">JSON</option>
-        <option value="xml">XML</option>
+        <option value=".json">JSON</option>
+        <option value=".xml">XML</option>
```

Comment:
Is it better not to have "." in the option value here, and instead add that "." later in js?